File: C-Sharp/C Sharp lektioner/Lektion 10 - Web/Models/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Lektion_10___Web.Models
{
    public class Person
    {
        string Name { get; set; }
        string Addresse { get; set; }
        int Alder { get; set; }
        string Information { get; set; }

        public Person(string navn, string addresse, int alder, string info)
        {
            Name = navn;
            Addresse = addresse;
            Alder = alder;
            Information = info;
        }
    }
}

File: Java/Semester 1 usorteret/Lektion13_Arraylister/src/eksempler/Person.java
package eksempler;

public class Person {
	
	private int alder;
	private String navn;
	
	public Person(String navn, int alder) {
		this.navn = navn;
		this.alder = alder;
	}
	
	public String getNavn() {
		return navn;
	}
	
	public int getAlder() {
		return alder;
	}
}
